dev/conductor/lib/codesign.dart

```diff
@@ -112,29 +112,33 @@ class CodesignCommand extends Command<void> {
           'the desired revision and run that version of the conductor.\n');
       revision = argResults![kRevision] as String;
     } else {
-      revision = (processManager.runSync(
+      revision = ((await processManager.run(
         <String>['git', 'rev-parse', 'HEAD'],
-        workingDirectory: framework.checkoutDirectory.path,
-      ).stdout as String).trim();
+        workingDirectory: (await framework.checkoutDirectory).path,
+      ))
+              .stdout as String)
+          .trim();
       assert(revision.isNotEmpty);
     }
 
-    framework.checkout(revision);
+    await framework.checkout(revision);
 
     // Ensure artifacts present
-    framework.runFlutter(<String>['precache', '--android', '--ios', '--macos']);
+    await framework
```

Comment:
this violates our style guide, either make this one line or line break after the open paren